cellbender/remove_background/consts.py

```diff
@@ -21,10 +22,17 @@
 TOTAL_DROPLET_DEFAULT = 25000
 
 # Fraction of the data used for training (versus testing).
-TRAINING_FRACTION = 1.
+TRAINING_FRACTION = 0.9
 
 # Size of minibatch by default.
 DEFAULT_BATCH_SIZE = 128
 
 # Fraction of totally empty droplets that makes up each minibatch, by default.
-FRACTION_EMPTIES = 0.5
\ No newline at end of file
+FRACTION_EMPTIES = 0.5
+
+# Prior on rho, the swapping fraction: the two concentration parameters alpha and beta.
+RHO_ALPHA_PRIOR = 18.
+RHO_BETA_PRIOR = 200.
+
+# Prior on epsilon, the RT efficiency concentration parameter [Gamma(alpha, alpha)].
+EPSILON_PRIOR = 500.
```

Comment:
I think that is a great idea, yes, I am experimenting with that now and I think it should be part of `0.2.1`.  I do think it makes sense mechanistically that epsilon is responsible for the dispersion in the empty droplet UMI counts, rather than differences in physical droplet size (which I think is pretty tightly controlled by the instrument).

Reply:
please make a github issue about it so we won't forget to take care of it later.